tasks/security_agent.py

```diff
@@ -10,19 +10,28 @@
 from subprocess import check_output
 
 from .utils import bin_name, get_gopath, get_build_flags, REPO_PATH, get_version, get_git_branch_name, get_go_version, get_git_commit, get_version_numeric_only
-from .build_tags import get_default_build_tags
+from .build_tags import get_build_tags, LINUX_ONLY_TAGS
+from .go import generate
 
 BIN_DIR = os.path.join(".", "bin", "security-agent")
 BIN_PATH = os.path.join(BIN_DIR, bin_name("security-agent", android=False))
 GIMME_ENV_VARS = ['GOROOT', 'PATH']
 
+DEFAULT_BUILD_TAGS = [
+    "netcgo",
+    "secrets",
+    "docker",
+    "kubeapiserver",
+    "kubelet",
+]
+
 @task
 def build(ctx, race=False, go_version=None, incremental_build=False,
-          major_version='7', arch="x64", go_mod="vendor"):
+          major_version='7',  python_runtimes='3', arch="x64", go_mod="vendor"):
```

Comment:
We should need any python reference here since the security agent don't use it.

Reply:
Removed, thank you.